core/templates/dev/head/pages/exploration-editor-page/exploration-editor-page.controller.ts

```diff
@@ -38,7 +25,10 @@ require(
   'attribution-guide.directive.ts');
 require(
   'components/forms/custom-forms-directives/select2-dropdown.directive.ts');
+require(
+  'components/forms/schema-based-editors/schema-based-editor.directive.ts');
 require('components/profile-link-directives/profile-link-text.directive.ts');
+require('directives/angular-html-bind.directive.ts');
```

Comment:
Probably unneeded.

Reply:
It is used in modals loaded in exploration-editor tab. I've moved the require there as well.